libcontainer/system/rlimit_go119.go

```diff
@@ -0,0 +1,24 @@
+//go:build go1.19
+
+package system
+
+import (
+	"sync/atomic"
+	"syscall"
+
+	_ "unsafe" // for go:linkname
+)
+
+//go:linkname syscallOrigRlimitNofile syscall.origRlimitNofile
+var syscallOrigRlimitNofile atomic.Pointer[syscall.Rlimit]
+
+// As reported in issue #4195, the new version of go runtime(since 1.19)
```

Comment:
Looks like this was also in the cherry-picked commit, but we should fix this comment to be in correct GoDoc format (on main at least)